examples/python_django/README.md

```diff
@@ -0,0 +1,49 @@
+# Django Example
```

Comment:
Is there a reason why this is called `python_django` and not `python-django`?

Reply:
yes, django doesn't accept `-` in the name :(